services/src/main/java/org/keycloak/theme/ResourceLoader.java

```diff
@@ -15,7 +15,12 @@ public static InputStream getResourceAsStream(String root, String resource) thro
         Path rootPath = Path.of("/", root).normalize().toAbsolutePath();
         Path resourcePath = rootPath.resolve(resource).normalize().toAbsolutePath();
         if (resourcePath.startsWith(rootPath)) {
-            URL url = classLoader().getResource(resourcePath.toString().substring(1));
+            if (File.separatorChar == '/') {
+                resource = resourcePath.toString().substring(1);
+            } else {
+                resource = resourcePath.toString().substring(2).replace('\\', '/');
+            }
+            URL url = classLoader().getResource(resource);
             return url != null ? url.openStream() : null;
```

Comment:
Perhaps we can call `getResourceAsStream()` here instead?
```suggestion
            return classLoader().getResourceAsStream(resource);
```

Reply:
Probably, but don't want to change unnecessary stuff now, and just focus on getting the current blocker bug fixed.